platform-server-kernel/src/main/java/uno/cod/platform/server/core/repository/TeamInvitationRepository.java

```diff
@@ -11,6 +11,6 @@
 
 public interface TeamInvitationRepository extends JpaRepository<TeamInvitation, TeamUserKey> {
     @Query("SELECT DISTINCT invitation FROM TeamInvitation invitation " +
-            "WHERE invitation.key.user.id = :userId")
+            "WHERE invitation.key.team.enabled = true AND invitation.key.user.id = :userId")
```

Comment:
remove all invites to a team, when disabling one team, and check before actually inviting if a team has been disabled in between, if yes, return an error and just remove the invite (if not already done)